### ISSUE TITLE
Choose platform

### ISSUE THREAD

**Issue by emmyft** [softwaresaved/eventure#1]
Explore platforms:
- Twine: http://twinery.org/
- Other choices?

**Comment by adrian-castravete** [comment]
I've tested all the “Story Formats” (which are like rendering engines) and not one of them supports images out of the box.
It mainly has to do with serving images from an HTML file without an HTTP server.
Hacking something I can make a temporary HTTP Server with
```
python -m SimpleHTTPServer
```
Which would open a server at port 8000, I can make it display an image. So accessing `http://localhost:8000` you'd be able to see the game. But this is only after you exported the game to a file.
Needless to say coding it would be painful.

A solution is that we could just ditch the idea of images.
Alternatively we could use a game engine and not necessarily make it a web application.

**Comment by adrian-castravete** [comment]
Oh, so it seems I did things wrongly, As @npch stated in another issue if we don't put the data in an _assets_ directory it's going to work.

**Comment by adrian-castravete** [comment]
Also note the the image, audio or other file would need to be copied in the `{Documents}/Twine/Stories` if a contributor wants to see/hear them while contributing.

**Comment by davclark** [comment]
For reference: https://vagrantcursor.wordpress.com/2018/01/02/tools-to-make-narrative-games/

There are definitely approaches that are more accessible for text-editing / git-like approaches!

**Comment by emmyft** [comment]
We settled on Twine.
